### DISCUSSION TITLE
How do I add my own treesitter rules

### DISCUSSION THREAD

**Discussion by SephVelut** [andymass/vim-matchup#259] [Q&A]
Rust matching angle brackets are not highlighted. How could I add my own treesitter queries for matchup to highlight matching angle brackets under cursor?

**Comment by andymass** [comment]
See https://github.com/andymass/vim-matchup/wiki/Supporting-a-new-language-(tree-sitter).

You'll want to edit this file https://github.com/andymass/vim-matchup/blob/master/after/queries/rust/matchup.scm

to add a new set of queries for type_arguments "<" and ">" as open and close.
